src/DocLinkChecker/Program.cs

```diff
@@ -195,6 +205,17 @@ private static void ProcessFiles(DirectoryInfo folder)
 
             foreach (FileInfo fi in files)
             {
+                if (!string.IsNullOrEmpty(options.SkipPattern))
+                {
+                    message.Verbose($"Skip pattern: {options.SkipPattern}");
+
+                    if (Regex.IsMatch(fi.FullName.ToLowerInvariant(), options.SkipPattern))
```

Comment:
why do you compare in lower invariant? If you are using a regex, you can handle this in the regex?

Reply:
you are right, removed